internal/factsengine/gatherers/cibadmin_test.go

```diff
@@ -101,6 +101,12 @@ func (suite *CibAdminTestSuite) TestCibAdminGather() {
 			Argument: "cib.not_found.crm_config",
 			CheckID:  "check3",
 		},
+		{
+			Name:     "primitives",
+			Gatherer: "cibadmin",
+			Argument: "cib.configuration.resources.primitive.0",
```

Comment:
I would have added also a case for `cib.configuration.resources.primitive` which returns a list, no blocker tho for me.

Reply:
@nelsonkopliku A list scenario is already covered within this argument value.
I chose specifically this primitive resource, as it has a single `nvpair`, which is in the `elementsToConvert` list. With that, I already cover that this unique element is converted to a list.

Reply:
All good then, thanks!